### PR TITLE
Block template react scss cleanup

### PR DESCRIPTION
**Pull request by luisbettencourt**
## 🌟 What is the purpose of this PR?

This PR strips down the contents of the scss file of `block-template-react` to include only the styles useful for the Hello World example.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203179076056186/1204069980884590/f) _(internal)_

## 🔍 What does this change?

- Removed variables;
- Made styles apply to classes instead of html elements;
- Removed styles not useful for the Hello World example.

## 🚀 Has this modified a publishable library?

This PR:

- [x] modifies an **npm**-publishable library and **I have added a changeset file(s)**
- [ ] modifies a **Cargo**-publishable library and **I have amended the version**
- [ ] modifies a **Cargo**-publishable library, but **it is not yet ready to publish**
- [ ] modifies a **block** that will need publishing
- [ ] does not modify any publishable blocks or libraries
- [ ] I am unsure / need advice

### CODE REVIEW

File: libs/block-template-react/src/base.module.scss [hunk 1]
```diff
@@ -1,251 +1,45 @@
...
 
+.block {
   /* –––––––––– Helper styles –––––––––– */
 
-  font-family: var(--font-family);
-  color: var(--primary-text-base);
-
-  /* Typography */
+  font-family: system-ui, "Inter", "Open Sans", "Helvetica Neue", "Arial",
+    sans-serif;
```
Comment (CiaranMn) [inline libs/block-template-react/src/base.module.scss:9]: We don't load these fonts so I'd just get rid of them
```suggestion
  font-family: system-ui, "Helvetica Neue", "Arial", sans-serif;
```
```

Comment (luisbettencourt) [reply libs/block-template-react/src/base.module.scss]: Done in 37b0a92d5d1fe8df92ccd12b4509ec0d28d6a8b3

Comment (CiaranMn) [reply libs/block-template-react/src/base.module.scss]: See suggestion above

Comment (luisbettencourt) [reply libs/block-template-react/src/base.module.scss]: I misunderstood, thank you. 7b2f14a38eb4968cb6b76d2560a15656eea3a637

**Review by CiaranMn — APPROVED** [review]

**Pull request merged**